interbtc-spec/docs/source/economics/incentives.rst

```diff
@@ -76,14 +76,14 @@ interBTC interest process
 -------------------------
 
 Fees paid in interBTC (on Issue, Redeem, and Replace) are forwarded to a fee pool.
-The fee pool then distributes the interBTC fees to all Vaults, Maintainers, and Collators according to a configurable distribution, and, if implemented, depending on the SLA score.
+The fee pool then distributes the interBTC fees to all Vaults and Collators according to a configurable distribution, and, if implemented, depending on the SLA score.
```

Comment:
Interbtc 1.5 document mentions neither collateral, nor maintainers. Why did you remove maintainers but not collators?

Reply:
This was agreed to remove in the specification, I'm not sure what to do with the Collators - this is possibly related to the token model.